msgraph-connector-demo/src/msgraph/wrappers/MsGraphRestWrapper.java

```diff
@@ -0,0 +1,41 @@
+package msgraph.wrappers;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+
+import com.microsoft.graph.CollectionOfMessage;
+import com.microsoft.graph.MicrosoftGraphMessage;
+
+import ch.ivyteam.ivy.environment.Ivy;
+
+public class MsGraphRestWrapper {
+  private final WebTarget restClient;
+
+  public MsGraphRestWrapper() {
+    restClient = Ivy.rest().client(UUID.fromString("007036dc-72d1-429f-88a7-ba5d5cf5ae58"));
+  }
+
+  public MsGraphRestWrapper pageSize(int top)
+  {
```

Comment:
keep in mind, that we no longer format like this since 9.3 or so, please keep the formatting consistent in this project